bundle/compliance/cis_gcp/rules/cis_1_15/rule.rego

```diff
@@ -0,0 +1,24 @@
+package compliance.cis_gcp.rules.cis_1_15
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.contains
+import future.keywords.if
+
+default is_valid = false
+
+duration = sprintf("%dh", [90 * 24]) # 90 days converted to hours
+
+is_valid if {
+	date := time.parse_rfc3339_ns(data_adapter.resource.data.createTime)
+	common.date_within_duration(date, duration)
+}
```

Comment:
Might be better to store in a different audit.
For example, Rule `1.7` will have the same logic, "Ensure User-Managed/External Keys for Service Accounts Are Rotated Every 90 Days or Fewer".